docs/UserGuide.md

```diff
@@ -750,28 +751,45 @@ Term | Explanation
 
 # 8. Command summary <a id="8-command-summary"></a>
 
+## Recipe-Related Command <a id="81-recipe-related-commands"></a>
+
 Features | Format, Examples
 --------|------------------
 **Add recipe** | `addR n/NAME i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]] c/CALORIES [img/IMAGE] inst/INSTRUCTION[. MORE INSTRUCTIONS] [t/TAG]...` <br> e.g. `addR n/salad i/lettuce, tomato, olive oil c/40 img/images/healthy1.jpg instr/Cook. Eat. t/fast t/easy`
 **List recipes** | `recipes`
 **Delete recipe** | `deleteR INDEX`<br> e.g. `deleteR 3`
 **Edit recipe** | `editR INDEX [n/NAME] [i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]]] [c/CALORIES] [img/IMAGE] [inst/INSTRUCTION[. MORE INSTRUCTIONS]] [t/TAG]...` <br> e.g. `editR 2 n/Apple salad i/apple`
-**Get edit recipe info** | `editR INDEX` <br> e.g. `editR 2`
-**Select recipe**| `selectR INDEX` <br> e.g. `selectR 3`
+**Get edit recipe info** | `editR INDEX`<br> e.g. `editR 2`
+**Select recipe** | `selectR INDEX`<br> e.g. `selectR 3`
 **Close recipe drawer**| `close`
-**Search for recipe** | `searchR [i/INGREDIENT [MORE_INGREDIENT]] [n/NAME [MORE_NAME]] [t/TAG [MORE_TAG]]` <br> e.g. `searchR i/lettuce tomato`, `searchR n/salad`, `searchR t/healthy`
+**Search for recipe** | `searchR [i/INGREDIENT [MORE_INGREDIENT]] [n/NAME [MORE_NAME]] [t/TAG [MORE_TAG]]`<br> e.g. `searchR i/lettuce tomato, searchR n/salad, searchR t/healthy`
```

Comment:
i think in this example put the ` ` around each command?

Reply:
okay!